internal/release/postinstall.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
```

Comment:
```suggestion
# Copyright The OpenTelemetry Authors
# Copyright Dynatrace LLC
```
since you made adaptions to it - same for the other files

Reply:
good catch, done!